lib/matrix.test.js

```diff
@@ -375,3 +376,16 @@ test('matrix (de)serialization', () => {
     data: m.data
   });
 });
+
+test('matrix copy', () => {
+  let m = new Matrix(5, 5);
+  m.randomize();
+
+  let n = m.copy();
+
+  expect(n).toEqual({
+    rows: m.rows,
+    cols: m.cols,
+    data: m.data
```

Comment:
will this work? Won't we have to check the values of the arrays themselves as opposed to the `data` reference?

Reply:
It does, as this does not pass:
```javascript
  let n = m.copy();
  m.add(1);

  expect(n).toEqual({
    rows: m.rows,
    cols: m.cols,
    data: m.data
  });
```
![schermafdruk_2018-03-10_18-22-24](https://user-images.githubusercontent.com/834728/37244896-2e72dfd6-2490-11e8-9dc7-eab7459f5ea6.png)

Reply:
Actually `expect(n).toEqual(m);` works as well 😆

Reply:
Oh cool! So Jest does something like a "deep" comparison behind the scenes?